recipes/biofilm/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "biofilm" %}
+{% set version = "0.0.94" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/biofilm-{{ version }}.tar.gz
+  sha256: b285d80da26aebdf41c7b4273c1b31717a035ff731169092a4ca1ac98b2cff3d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: True  # [win or osx or py<38]
```

Comment:
This package should be `noarch: python`. It doesn't matter that it is uninstallable on Windows and OSX right now. If auto-sklearn or other packages become available for Windows and OSX later, it will become installable then without needing to rebuild your package.

https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=noarch#noarch-python

Reply:
@carterbox it looks like it does matter as the auto-testing complains now.